### DISCUSSION TITLE
DRT Parser?

### DISCUSSION THREAD

**Discussion by ehknight** [explosion/spaCy#580] [New Features & Project Ideas]
Are there any plans to implement a DRT (discourse representation theory) system in spaCy, perhaps similar to C&C Tools+Boxer?

**Comment by honnibal** [comment]
No plans at the moment. I did my PhD on the C&C system, and I've done a lot of work on noun phrase semantics, so I'd like to have something like this. I don't have a design in mind though. Do you want to propose something?
